### ISSUE TITLE
A11y: alt attribute of header logo and link title must not be equal

### ISSUE THREAD

**Issue by tieupmedia** [shopware/platform#10527]
### Shopware Version

6.6.10.5 / 6.7.0.0-rc5

### Affected area / extension

Platform(Default)

### Actual behaviour

The `alt` attribute of the logo image and the link `title` are equal and can't be set different:

![Image](https://github.com/user-attachments/assets/6d8135b3-28cf-488f-8b9b-cd92e3453bfc)

### Expected behaviour

It should be possible to customize each attribute separately to avoid duplication and target their different purposes:

1. title of the link:
  - describe the action
  - e.g. “Go to homepage”
2. alt-text of the logo image
  - describe the image, text replacement for the image
  - at least "Logo of ###company name###"
  - even better when including how the logo looks like or any subline that is shown in the image

### How to reproduce

1. Open any Shopware shop and inspect the html output in the browsers console
2. Or see the relevant twig code: https://github.com/shopware/shopware/blob/v6.6.10.5/src/Storefront/Resources/views/storefront/layout/header/logo.html.twig

### Definition of Done

- [ ] An automated test covers the fix.
- [ ] Integration/E2E testing in staging is done.
- [ ] Changelog markdown file created or updated in /changelog/_unreleased directory.
- [ ] Developer documentation written/updated.
- [ ] End user and/or developer documentation written or updated with clarification.
- [ ] Regression test created to ensure bug does not reappear, and automated regression tests pass.
- [x] Code changes resolve the reported bug without introducing new issues.

**Comment by tobiasberge** [comment]
Hi @tieupmedia,

sorry for the late reply. I was looking into this and I would propose removing the title attribute altogether instead of adding another snippet key.

https://github.com/shopware/shopware/pull/18989

I have investigated a bit and it looks like it is considered a good practice to have a "plain" link without title and let the link be described by the alt attribute of the image itself. This is suggested by W3C: https://www.w3.org/WAI/tutorials/images/functional/#example-1-image-used-alone-as-a-linked-logo

From what I have read it is also not good practice to use the title attribute as a descriptor for the screenreader because it is not supported well (To be fair, for me it worked). Also, according to [this](https://www.w3.org/TR/WCAG20-TECHS/H33.html) the title should not be used as a label for a link, but it should be used as additional info about the link that already has a label. At least thats how I understand it.

Alternatively, we could add an `aria-label` on the link and a different alt attribute. I have also seen this on some pages. However, it looks like the `aria-label` overrides the alt attribute then anyway so it is never read by a screenreader. 

If there should be more info about the logo itself, one could still modify the `header.logoLink` snippet like e.g. `Demoshop Logo - Go to homepage`.

Let me know if I'm overlooking something. Maybe you have a usecase for this I'm not seeing.
